manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -257,19 +257,19 @@ properties:
       bbs:
         api_location: bbs.service.cf.internal:8889
         ca_cert: (( grab secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
-        require_ssl: false
+        client_cert: (( grab secrets.bbs_client_cert  ))
+        client_key: (( grab secrets.bbs_client_key  ))
+        require_ssl: true
       dropsonde_port: (( grab properties.metron_agent.dropsonde_incoming_port ))
 
     bbs:
       active_key_label: keylabel1
       encryption_keys:
         - label: keylabel1
-          passphrase: "a secure passphrase"
-      ca_cert: (( grab secrets.consul_ca_cert ))
-      server_cert: (( grab secrets.consul_server_cert ))
-      server_key: (( grab secrets.consul_server_key ))
+      passphrase: "a secure passphrase"
```

Comment:
I think this `passphrase` has been unindented by accident. 

Anyway, do we use the `encription_keys` at all? More info [in the spec](https://github.com/cloudfoundry-incubator/diego-release/blob/develop/jobs/bbs/spec#L35) and [the template](https://github.com/cloudfoundry-incubator/diego-release/blob/7045d0f81845ff1e105c75163d9c901f2d4067ef/jobs/bbs/templates/bbs_ctl.erb#L85)